.github/workflows/example-basic.yml

```
@@ -20,7 +20,8 @@ jobs:
         # the parameters below are only necessary
         # because we are running these examples in a monorepo
         with:
-          working-directory: examples/basic
+          working-directory: examples/v9/basic
+          config-file: cypress.json
```

@robertguss Might be a good idea to add a comment at these config-file override for context
```suggestion
          config-file: cypress.json # Need to specify cypress.json for Cypress v9 and lower
```

---

Not sure if we need to as this is documented in the readme. Also each config file has a comment which separates v9 jobs from v10

---

I saw, was just a suggestion.